gnat2goto/driver/ureal_to_binary.adb

```diff
@@ -193,10 +189,14 @@ package body Ureal_To_Binary is
                      First_One := False;
                   end if;
                else
-                  Result (Fraction_Bit) := Bit;
-                  Fraction_Bit := Fraction_Bit + 1;
                   Exponent := Exponent + 1;
+                  --  check that we do not start writing past the fraction part
```

Comment:
Would be good to explain the implication/result if you *would* have written past the fraction part

Reply:
The fraction bits are what we're trying to write here, so even if the fraction bit wasn't the end of the string we're writing (which it is), it'd still be an unsound thing to do

Reply:
Done.